apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-content-palette/dot-content-palette.component.scss

```diff
@@ -36,23 +49,20 @@
 .dot-content-palette__items {
 
     overflow: hidden auto;
+    display: grid;
```

Comment:
Why grid to something that is just a icon and text aligned vertically?

Reply:
any different make them looks incorrect. but grid can be used to align two rows as well.